tutorials/writing-tutorials.md

```diff
@@ -45,6 +45,21 @@ For each tutorial:
    - (optional) Include hints in the code skeletton to highlight where the students should complete the code and what python functions they could use to complete it (e.g.: '#Hint: use the function `np.exp()` to exponentiate' )
    - (optional) A sample output of what the correct output of the exercise should look like. In order to prevent students from focussing on reproducing exactly the plots/expected-outputs rather than understanding the core concepts, we provide them with the plotting functions, and use XKCD style for the expected sample outputs.
 4. Create sample answers for each tutorial, and so that the technical team can check the tutorials, and to facillitate training for the TAs prior to the summer school.
+5. Generate student versions that exclude sample answers and other auxiliary cells. Follows the instructions below to execute this step automatically.
```

Comment:
I think we should tell them just to generate the TA version in this doc. Otherwise they're going to wait until the last minute to send their super polished notebooks. I would rather see a rough cut very early on.

Reply:
Sounds good. If you are ok with content, please merge this one first and you could submit the initial split writing-tutorials.md / polishing-tutorials.md?